code/modules/reagents/reagent_containers/food/snacks.dm

```diff
@@ -435,6 +441,7 @@
 	nutriment_amt = 2
 	nutriment_desc = list("chocolate" = 5)
 	preloaded_reagents = list("sugar" = 2, "coco" = 2)
+	taste_tag = COCO_FOOD
```

Comment:
This clearly should be "sweet" food as well. And all other food should have multiple tags.